core/src/main/java/site/ycsb/workloads/CoreWorkload.java

```diff
@@ -611,7 +611,7 @@ private String buildDeterministicValue(String key, String fieldkey) {
    */
   @Override
   public boolean doInsert(DB db, Object threadstate) {
-    int keynum = keysequence.nextValue().intValue();
+    long keynum = keysequence.nextValue().intValue();
```

Comment:
```suggestion
    long keynum = keysequence.nextValue();
```

Drop the `intValue()`?  `nextValue()` is returning back a `Long` and it will be unboxed.

Reply:
@joshelser My earlier patch was missing the change to return the long value. Fixed the patch to get the long type from the Number returned by nextValue().